app/src/main/java/org/mozilla/fenix/compose/LightDarkPreview.kt

```diff
@@ -0,0 +1,8 @@
+package org.mozilla.fenix.compose
```

Comment:
to prevent the root `.../compose/` directory from becoming too bloated, let's move this to `.../compose/annotation`. We'll add any more similar annotation classes there in the future.

Reply:
sounds good, will make the new directory for annotation